documentation/book/ref-kafka-authentication.adoc

```diff
@@ -6,17 +6,18 @@
 = Authentication
 
 Authentication is configured as part of the xref:assembly-configuring-kafka-broker-listeners-{context}[listener configuration] in the `authentication` property.
-When the `authentication` property is missing, no authentication will be enabled on given listener.
-The authentication mechanism which will be used is defined by the `type` field.
+The authentication mechanism is defined by the `type` field.
 
-The supported authentication mechanisms are: 
+When the `authentication` property is missing, no authentication is enabled on a given listener.
+
+Supported authentication mechanisms:
```

Comment:
```suggestion
Supported authentication mechanisms are:
```

Feel free to ignore this ... but it sounds better to me with the `are` added.

Reply:
When used with a list the colon is the 'are', so I don't think we need it.